addons/crm/models/crm_team_member.py

```diff
@@ -26,16 +26,34 @@ class TeamMember(models.Model):
     lead_month_count = fields.Integer(
         'Leads (30 days)', compute='_compute_lead_month_count',
         help='Lead assigned to this member those last 30 days')
+    lead_day_count = fields.Integer(
```

Comment:
Do not compute both fields through the same method. lead_month_count is not necessary to do lead assignation, so it's better to not even compute it. (as it will count all leads for a month). In the other end, lead_day_count is not displayed in the UI; no need to compute it either.

Instead, I would do two computes.